inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp

```diff
@@ -510,36 +499,41 @@ InferenceEngine::details::CNNLayerCreator::CNNLayerCreator(const std::shared_ptr
 
     });
 
-    addSpecificCreator({"PriorBox"}, [](const std::shared_ptr<::ngraph::Node>& node,
-                                       const std::map<std::string, std::string> params) -> CNNLayerPtr {
-        // Todo (itikhono): replace the message after supporting constants as outputs in plugins
-//        THROW_IE_EXCEPTION << "PriorBox operation has a form that is not supported." << node->get_friendly_name()
-//                           << " should be replaced by constant during constant folding.";
-        THROW_IE_EXCEPTION << "PriorBox operation has a form that is not supported." << node->get_friendly_name()
-                           << " should be converted to PriorBoxIE operation";
-        return nullptr;
-    });
-
-    addSpecificCreator({"PriorBoxClustered"}, [](const std::shared_ptr<::ngraph::Node>& node,
-                                       const std::map<std::string, std::string> params) -> CNNLayerPtr {
-        // Todo (itikhono): replace the message after supporting constants as outputs in plugins
-//        THROW_IE_EXCEPTION << "PriorBoxClustered operation has a form that is not supported." << node->get_friendly_name()
-//                           << " should be replaced by constant during constant folding.";
-        THROW_IE_EXCEPTION << "PriorBoxClustered operation has a form that is not supported." << node->get_friendly_name()
-                           << " should be converted to PriorBoxClusteredIE operation";
-        return nullptr;
-    });
-
     addSpecificCreator({"NonMaxSuppressionIE"}, [](const std::shared_ptr<::ngraph::Node>& node,
-                                                 const std::map<std::string, std::string> params) -> CNNLayerPtr {
+                                                 const std::map<std::string, std::string>& params) -> CNNLayerPtr {
         LayerParams attrs = {node->get_friendly_name(), "NonMaxSuppression", details::convertPrecision(node->get_output_element_type(0))};
         auto res = std::make_shared<InferenceEngine::NonMaxSuppressionLayer>(attrs);
         res->params = params;
         return res;
     });
 
+    addSpecificCreator({ "Broadcast", "Convolution", "ConvolutionBackpropData", "Gather", "GatherTree",
+                         "GroupConvolution", "GroupConvolutionBackpropData", "GRUCell", "GRUSequence", "HardSigmoid",
+                         "Interpolate", "LRN", "LSTMCell", "LSTMSequence", "NonMaxSuppression", "NormalizeL2",
+                         "RNNCell", "RNNSequence", "OneHot", "Pad", "PriorBoxClustered", "PriorBox", "Proposal",
+                         "Selu", "Swish", "Tile", "TopK"},
+            [](const std::shared_ptr<::ngraph::Node>& node, const std::map<std::string, std::string>& params)
+            -> CNNLayerPtr {
+        std::string type_name = node->get_type_name();
+        std::map<std::string, std::string> exceptions = { {"Broadcast", "Tile"}, {"Interpolate", "Interp"},
```

Comment:
Maybe we can add a macro which will register such creators.
For example:
```
CONVERSION_REQUIRED_CREATOR("Broadcast", "BroadcastIE");
...
CONVERSION_REQUIRED_CREATOR("NormalizeL2", "NormalizeIE");
```

Reply:
For some of these operations we use NodeConverter, for the other part we use Creators, how should this macro work?

Reply:
Added. Any suggestions for changing the macro name?